### ISSUE TITLE
[Bug] Phpmyadmin config not readable Debian 12 new install

### ISSUE THREAD

**Issue by jlguerrerosanz** [hestiacp/hestiacp#4405]
### Describe the bug

After a new install, I create a domain with a database.

When I try to access phpmyadmin i encounter the following error in the browser:

"phpMyAdmin - Error
Existing configuration file (/etc/phpmyadmin/config.inc.php) is not readable."

so I
ls -l /etc/phpmyadmin/config.inc.php

and get this:
-rw-r----- 1 root **hestiamail** 980 May 27 15:06 **/etc/phpmyadmin/config.inc.php**

since permissions are 640 I changed them to 644 and it works again.

Can you reproduce the error? Is this the right fix or should I change the owner instead?

### Tell us how to replicate the bug

New Debian 12 install
Create any user
Create database
visit subdomain.domain.com/phpmyadmin

### Which components are affected by this bug?

Control Panel Web Interface, Database (MariaDB, MySQL, PostgreSQL)

### Hestia Control Panel Version

Latest

### Operating system

Debian 12

**Comment by hestiacn** [comment]
Use the chmod 644 /etc/phpmyadmin/config.inc.php command to run successfully

**Comment by jlguerrerosanz** [comment]
> Use the chmod 644 /etc/phpmyadmin/config.inc.php command to run successfully

Thank you very much for your response.

It makes no sense that the ownership of the file is root:hestia-mail

**This has to be a bug**.

I corrected the ownership again and reverted permissons to 640.

**Comment by hestiacn** [comment]
This does not affect usage. Is it possible to set the hestia-mail user group just to distinguish hestia web users, that is, to see the user group to know that the file or folder is installed by hestia?

**Comment by jlguerrerosanz** [comment]
Of course. This does not affect usage. It is about security. I prefer 640 to 644. 

It is also about a file that has been installed by hestiacp that has the wrong permissions just out of the box.

That's it. I am telling you so you can check if I am the only one getting this error or maybe someone else is also getting the same behavior and it is worth your time to fix this.

Thank you again for your time and your wonderful work.

**Comment by sinarkreasi** [comment]
Thanks **"chmod 644 /etc/phpmyadmin/config.inc.php"** it works with Ubuntu 22.04
